modules/acid/src/main/ts/ephox/acid/api/colour/RgbaColour.ts

```diff
@@ -121,7 +121,7 @@ const fromString = (rgbaString: string): Option<Rgba> => {
 
 const toString = (rgba: Rgba): string => `rgba(${rgba.red},${rgba.green},${rgba.blue},${rgba.alpha})`;
 
-const red = rgbaColour(255, 0, 0, 1);
+const redColour = rgbaColour(255, 0, 0, 1);
```

Comment:
I did this rename manually to fix the aliasing, this was not an autofix.

Reply:
ha, and here I thought it was being clever